brush-parser/Cargo.toml

```diff
@@ -20,11 +20,13 @@ bench = false
 [features]
 fuzz-testing = ["dep:arbitrary"]
 debug-tracing = ["peg/trace"]
+fancy-diagnostics = ["dep:miette"]
```

Comment:
Thanks for clarifying that this is basically just for the example. Would you mind adding a comment here explaining that this is generally not needed, and instead the consuming app can/should enable it itself?

Reply:
This is to make miette optional, so people that do not want to depend on miette they can just use the support provided by thiserror alone.